cmake/Modules/ThirdpartyToolchain.cmake

```diff
@@ -118,6 +118,7 @@ if ("${CMAKE_RAY_LANG_PYTHON}" STREQUAL "YES")
 
     # PYARROW_PARALLEL= , so it will add -j to pyarrow build
     set(pyarrow_ENV
+      "SETUPTOOLS_SCM_PRETEND_VERSION=0.11.1-RAY"
```

Comment:
So does this need to be updated every time we update arrow? If so, we should include a prominent comment in https://github.com/ray-project/ray/blob/8674606e2687074b043129f7e743d6522d015c90/cmake/Modules/ArrowExternalProject.cmake#L17

explaining that people need to update this code.

Reply:
fixed